pkg/storage/enginepb/mvcc3.proto

```diff
@@ -207,6 +207,12 @@ message MVCCValueHeader {
   // ImportEpoch identifies the number of times a user has called IMPORT
   // INTO on the table this key belongs to when the table was not empty.
   uint32 import_epoch = 4;
+  
+  // OriginID identifes the original cluster that wrote this key during Logical
+  // Data Replication (LDR). 0 identifies a local write and the default value
+  // when LDR is not being run, 1 identifies a remote write, and 2+ are reserved
+  // to identify remote clusters.
+  uint32 origin_id = 5  [(gogoproto.customname) = "OriginID"];
 }
```

Comment:
we should `// Next ID` these in the next PR (for timestamps)

Reply:
Done.